provider/storage_benchmark.py

```diff
@@ -227,6 +227,17 @@ def clean(self, timeout=1800, force=False):
         if self.env_files:
             self._remove_env_files()
 
+    @classmethod
```

Comment:
I believe `staticmethod` is better than `classmethod`

Reply:
Thanks, have updated.